### ISSUE TITLE
RUN TA: sign.co2mpas.key 

### ISSUE THREAD

**Issue by sapofra** [JRCSTU/co2wui#125]
@stefanocorsi: Run TA should or should not require sign.co2mpas.key to run? 
sign.co2mpas.key should be generated automatically (@vinci1it2000 correct?), but at the moment I had to manually copy it in the folder otherwise the button RUN TA does not activate.

**Comment by vinci1it2000** [comment]
@sapofra Yes. If the `sign.co2mpas.key` is not in the DICE_KEYS folder, it will be automatically generated.

**Comment by stefanocorsi** [comment]
In the version of Co2mpas that was used for the build that you're testing the sign.co2mpas.key whas not yet automatically generated. @vinc1it2000 is it working now?

**Comment by vinci1it2000** [comment]
Yes, but it was working also before with co2mpas_dice v3.2.0.
